exercises/creating-a-pull-request/ABOUT_ME.md

```diff
@@ -0,0 +1,7 @@
+# About me
+
+## Shakti Prasad Nanda - name
+
+Hi I am **Shakti** !
```

Comment:
```suggestion
Hi, I am **Shakti**!
```

Reply:
I have added the missing _comma_ after Hi.